### PR TITLE
update: Add the meaning of TAILS in the OS recommendation page

### PR DESCRIPTION
**Pull request by rollsicecream**
Changes proposed in this PR:

- update: Add the meaning of TAILS in the OS recommendation page

<summary>

- [x] I agree to the terms listed below:
  <details><summary>Contribution terms (click to expand)</summary>
  1) I am the sole author of this work.
  2) I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
  3) I have disclosed any relevant conflicts of interest in my post.
  4) I agree to the Community Code of Conduct.
  </details>

### CODE REVIEW

File: docs/desktop.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ Whonix is best used [in conjunction with Qubes](https://whonix.org/wiki/Qubes/Wh
 
 ![Tails logo](assets/img/linux-desktop/tails.svg){ align=right }
 
-**Tails** is a live operating system based on Debian that routes all communications through Tor, which can boot on on almost any computer from a DVD, USB stick, or SD card installation. It uses [Tor](tor.md) to preserve privacy and anonymity while circumventing censorship, and it leaves no trace of itself on the computer it is used on after it is powered off.
+**Tails (or The Amnesic Incognito Live System)** is a live operating system based on Debian that routes all communications through Tor, which can boot on on almost any computer from a DVD, USB stick, or SD card installation. It uses [Tor](tor.md) to preserve privacy and anonymity while circumventing censorship, and it leaves no trace of itself on the computer it is used on after it is powered off.
```
Comment (jonaharagon) [inline docs/desktop.md:160]: I know that this is _technically_ true, but can you find any documentation on tails.net which explains the meaning of Tails? I suspect this is a similar situation to "Tor" where the name used to stand for something, but they no longer _officially_ use this meaning. If they don't say "The Amnesic Incognito Live System" anymore this info feels irrelevant.

**Review by jonaharagon — CHANGES_REQUESTED** [review]

File: docs/desktop.md (hunk 1 above)
Comment (jonaharagon) [reply docs/desktop.md:160]: I am not seeing any references to it newer than ~2015 which makes this pretty dated.

Comment (rollsicecream) [reply docs/desktop.md:160]: Sure. But there are other resources online that has that denomination like Wikipedia, Distrowatch, the Linux wiki in Fandom. It's just good to know that Tails means (or meant) that. It's just general knowledge.

Comment (jonaharagon) [reply docs/desktop.md:160]: > It's just good to know that Tails means (or meant) that.

...but why? I don't really follow what benefit this knowledge would have when sharing Tails with someone new.

Comment (rollsicecream) [reply docs/desktop.md:160]: Yeah..I just rethought a little bit and you're right. My bad.
